nitro-protocol/test/contracts/libraries/StrictTurnTaking/StrictTurnTaking.test.ts

```diff
@@ -0,0 +1,172 @@
+import {BigNumber, Contract, Wallet} from 'ethers';
+import {it} from '@jest/globals';
+import {expectRevert} from '@statechannels/devtools';
+
+import testStrictTurnTakingArtifact from '../../../../artifacts/contracts/test/TESTStrictTurnTaking.sol/TESTStrictTurnTaking.json';
+import {
+  getCountingAppContractAddress,
+  getRandomNonce,
+  getTestProvider,
+  setupContract,
+} from '../../../test-helpers';
+import {TESTStrictTurnTaking} from '../../../../typechain-types';
+import {Channel, getFixedPart, getVariablePart, Outcome, State} from '../../../../src';
+import {
+  INVALID_NUMBER_OF_PROOF,
+  SIGNED_BY_NON_MOVER,
+  TOO_MANY_PARTICIPANTS,
+} from '../../../../src/contract/transaction-creators/revert-reasons';
+import {RecoveredVariablePart} from '../../../../src/contract/state';
+import {getSignedBy} from '../../../../src/bitfield-utils';
+const provider = getTestProvider();
+let StrictTurnTaking: Contract & TESTStrictTurnTaking;
+
+const chainId = process.env.CHAIN_NETWORK_ID;
+const challengeDuration = 0x1000;
+const asset = Wallet.createRandom().address;
+const defaultOutcome: Outcome = [{asset, allocations: [], metadata: '0x'}];
+const appDefinition = getCountingAppContractAddress();
+const participants = ['', '', ''];
+const wallets = new Array(3);
+
+// Populate wallets and participants array
+for (let i = 0; i < 3; i++) {
+  wallets[i] = Wallet.createRandom();
+  participants[i] = wallets[i].address;
+}
+
+beforeAll(async () => {
+  StrictTurnTaking = setupContract(
+    provider,
+    testStrictTurnTakingArtifact,
+    process.env.TEST_STRICT_TURN_TAKING_ADDRESS
+  ) as Contract & TESTStrictTurnTaking;
+});
+
+let channelNonce = getRandomNonce('StrictTurnTaking');
+beforeEach(() => (channelNonce += 1));
+
+describe('isSignedByMover', () => {
+  const accepts1 = 'should not revert when signed only by mover';
+
+  const reverts1 = 'should revert when not signed by mover';
+  const INVALID_SIGNED_BY = SIGNED_BY_NON_MOVER;
```

Comment:
Why introduce this alias?

Reply:
It was removed in the following commits.